pkgs/development/python-modules/lizard/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "lizard";
+  version = "1.17.10";
+  disabled = pythonOlder "3.6";
```

I would suggest to limit it to Python releases which are not EOL. 

```suggestion
  disabled = pythonOlder "3.7";
```

---

Great idea! I'll update